### ISSUE TITLE
NameError: uninitialized constant Postmaster::PermissionError

### ISSUE THREAD

**Issue by fotinakis** [peterruggero/postmaster-ruby#11]
In lib/postmaster.rb:

```
def self.permission_error(error, rcode, rbody, error_obj)
    PermissionError.new(error[:message], error[:param], error[:code], rcode, rbody, error_obj)
```

But there isn't a `PermissionError` defined, so you get:

`NameError: uninitialized constant Postmaster::PermissionError`

**Comment by sargo** [comment]
Fixed in https://github.com/postmaster/postmaster-ruby/pull/13, released in [tag v1.3.2](https://github.com/postmaster/postmaster-ruby/releases/tag/v1.3.2) and pushed to [rubygems](http://rubygems.org/gems/postmaster/versions/1.3.2).

**Comment by fotinakis** [comment]
Thanks!
